blackbox/docs/admin/auth/hba.rst

```diff
@@ -19,9 +19,20 @@ See: :ref:`applying-cluster-settings`.
 
 .. NOTE::
 
+   The ``crate.yml`` that is shipped with CrateDB explicitly enables host based
+   authentication and defines a set of sane rules, which take affect in case
+   HBA and Enterprise features are enabled.
+
    Host Based Authentication is an
    :ref:`enterprise feature <enterprise_features>`.
 
+.. TIP::
+
+   If you want to use Enterprise features, but do not require authentication,
+   you need to set ``auth.host_based.enabled`` to ``false``. However,
+   additionally you still need to provide the user ``crate`` when connecting to
```

Comment:
afaik this is not true when using HTTP protocol, maybe we should mention?

Reply:
Pushed a fixup which I hope clarifies not required user when using http.

Reply:
lgtm, thx.